src/haddock/libs/libplots.py

```diff
@@ -709,171 +709,110 @@ def report_plots_handler(plots, shared_xaxes=False, shared_yaxes=False):
     return fig
 
 
-def _fix_uncluster_rank(row):
-    rank = row
-    if rank == "-":
-        rank = "Unclustered"
-    return rank
+def find_best_struct(df: pd.DataFrame, max_best_structs: int = 4) -> pd.DataFrame:
+    """Find best structures for each cluster.
 
+    Args:
+        df: DataFrame of capri_ss.tsv
+        max_best_structs: The maximum number of best structures to return.
 
-def find_best_struct(ss_file, number_of_struct=10):
+    Returns:
+        DataFrame of best structures with
+        `cluster_id` and `best<model-cluster-ranking>` columns
+        and empty strings for missing values.
     """
-    Find best structures.
+    # capri_ss.tsv has a column named "cluster-id" 
+    # capri_clt.tsv has a column named "cluster_id"
+    # rename here to make the merge easier
+    df.rename(columns={"cluster-id": "cluster_id"}, inplace=True)
```

Comment:
It would be nice if caprieval module would use consistent naming between both tsv files.

@mgiulini I have added the otther cluster and the PR is ready for review.

Reply:
good point!